boards/arm/efm32gg_stk3701a/doc/index.rst

```diff
@@ -0,0 +1,203 @@
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efm32gg_stk3701a/efm32gg_stk3701a_defconfig``
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFM32GG11 SoC has nine GPIO controllers (PORTA to PORTI), all of which are
+currently enabled for the EFM32GG-STK3701A board.
+
+In the following table, the column **Name** contains pin names. For example, PE2
+means pin number 2 on PORTE, as used in the board's datasheets and manuals.
+
++-------+-------------+-------------------------------------+
+| Name  | Function    | Usage                               |
++=======+=============+=====================================+
+| PH10  | GPIO        | LED0 red                            |
++-------+-------------+-------------------------------------+
+| PH11  | GPIO        | LED0 green                          |
++-------+-------------+-------------------------------------+
+| PH12  | GPIO        | LED0 blue                           |
++-------+-------------+-------------------------------------+
+| PH13  | GPIO        | LED1 red                            |
++-------+-------------+-------------------------------------+
+| PH14  | GPIO        | LED1 green                          |
++-------+-------------+-------------------------------------+
+| PH15  | GPIO        | LED1 blue                           |
++-------+-------------+-------------------------------------+
+| PC8   | GPIO        | Push Button PB0                     |
++-------+-------------+-------------------------------------+
+| PC9   | GPIO        | Push Button PB1                     |
++-------+-------------+-------------------------------------+
+| PE1   | GPIO        | Board Controller Enable             |
+|       |             | EFM_BC_EN                           |
++-------+-------------+-------------------------------------+
+| PH4   | UART_TX     | UART TX Console VCOM_TX US0_TX #4   |
++-------+-------------+-------------------------------------+
+| PH5   | UART_RX     | UART RX Console VCOM_RX US0_RX #4   |
++-------+-------------+-------------------------------------+
+| PI4   | I2C_SDA     | SENSOR_I2C_SDA I2C2_SDA #7          |
++-------+-------------+-------------------------------------+
+| PI5   | I2C_SCL     | SENSOR_I2C_SCL I2C2_SCL #7          |
++-------+-------------+-------------------------------------+
+
+
+System Clock
+============
+
+The EFM32GG11 SoC is configured to use the 50 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFM32GG11 SoC has six USARTs, two UARTs and two Low Energy UARTs (LEUART).
```

Comment:
Do we need to know which UART is assigned to the console?

Reply:
Added (USART4 in this case)